### PR TITLE
Vue3 Compiler

### PR DESCRIPTION
**Pull request by mggower**
This release will include support for typescript in vue & scss for styling. Fixes other small bugs with compiler and improves error handling

### CODE REVIEW

File: App.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      border="0"
+      width="450"
+      height="450"
+    />
+    <HelloVno msg="you are building: your project with vno" />
+  </div>
+</template>
+
+<script>
+import HelloVno from './HelloVno.js';
```
Comment (RivierGrullon) [inline App.vue:15]: shouldn't this be ".vue" instead of ".js"?

**Review by andrew-rehrig — APPROVED** [review]
changes approved

**Comment by andrew-rehrig** [comment]
ready to be merged

File: App.vue (hunk 1 above)
Comment (mggower) [reply App.vue:15]: Yeah it should be .vue

right now vno depends on the components property in the object to locate dependencies, and independently traverses the file structure to locate .vue files.
its not a breaking error unless the file itself is mislabeled.
we are currently trying to implement tools to save external imports into the final build, and this was likely overlooked

**Pull request merged**